Telereso/Classes/Telereso.swift

```diff
@@ -60,15 +62,21 @@ public struct Telereso{
         }
         remoteConfig.configSettings = settings!
         
-        
-        fetchResource(){ (shouldUpdate) -> Void in
-            if(shouldUpdate){
-                self.log("Fetched new data")
-                self.initMaps()
+        self.remoteConfig.activate { changed, error in
+            initMaps()
+            if !waitFetch {
+                completionHandler?()
+            }
+            fetchResource(){ (shouldUpdate) -> Void in
```

Comment:
let's put this out side the activate block, and before as well

Reply:
Ok